searx/settings.yml

```diff
@@ -1192,6 +1192,23 @@ engines:
       require_api_key: false
       results: JSON
 
+  - name: mullvad_leta
```

Comment:
The engine should be commented if it doesn't work out of the box.

Reply:
Thanks, after testing I'll do this and resolve the comment